docs/zh/commercial/cloud.md

```diff
@@ -3,51 +3,60 @@ toc_priority: 1
 toc_title: 云
 ---
 
-# ClickHouse 云服务提供商 {#clickhouse-cloud-service-providers}
+# ClickHouse云服务提供商 {#clickhouse-cloud-service-providers}
```

Comment:
I think it's better to add space between Chinese and English.

Refer: https://github.com/vinta/pangu.js